README.md

```diff
@@ -15,18 +15,18 @@ See the **[Quickstart guide](https://www.cloudquery.io/docs/quickstart)** for in
 
 ## Why CloudQuery?
 
-- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with the CloudQuery SDK.
-- **Blazing fast**: CloudQuery's concurrency system leverages the excellent Go concurrency model with light-weight Goroutines.
-- **Database agnostic**: CloudQuery can store your configuration in any supported destination such as a database, data lake, or streaming platform for further analysis.
-- **Pre-built queries**: CloudQuery maintains a rich set of SQL queries and Grafana dashboards for asset inventory, CSPMs, Security & Compliance, and Cost use-cases.
-- **Raw access to data**: Decouple data ingestion and have raw access to your data, with which you can build your own security stack and re-use
-  best-of-breed tools for querying (SQL,…), transformation (dbt,…), and visualization (Grafana, Preset, Metabase, PowerBI, …).
+- **Open source**: Extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
```

Comment:
I read up about it earlier, and `extensible` is commonly used in the software world... I think it's fine, maybe we can add a Vale exception